packages/kit/src/views/Swap/components/ReceivingTokenInput/index.tsx

```diff
@@ -412,25 +414,29 @@ const TokenInput: FC<TokenInputProps> = ({
     } = arbRebateData;
     const amountParsed = new BigNumber(amount)
       .shiftedBy(-decimals)
-      .decimalPlaces(4, BigNumber.ROUND_DOWN)
+      .decimalPlaces(2, BigNumber.ROUND_DOWN)
       .toFixed();
+    const isSvg = logoURI.toLowerCase().endsWith('.svg');
```

Comment:
```suggestion
function isSvgImageUrl(url) {
  try {
    return new URL(url).pathname.toLowerCase().endsWith('.svg');
  } catch (error) {
    console.error("Invalid URL:", error.message);
    return false;
  }
}
```

如果 Url 是携带 query 参数的呢，这样判断是不准确的

```
https://cryptologos.cc/logos/arbitrum-arb-logo.svg?v=029
https://example.com/image.svg?size=64
https://example.com/image.svg?width=64&height=64
```